dashboard/apps/prototype/migrations/0011_merge.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
will the `--fake` option help with this? i guess there is no big issue and the only downside is that the migration files in `develop` are already on prefix `0011`, with this merged in, we will get two `0009` and `0011`.